.github/workflows/ci.yml

```diff
@@ -8,98 +8,57 @@ env:
 jobs:
   build:
     name: build
-    runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        jdk: [8, 11, 14]
+        jdk: [8, 11]
```

Comment:
JDK14 support? Temurin doesn't have an image for it

Reply:
@scrawfor99 Could you look into this?  My current thinking is that we will merge it after this PR goes in - let me know what you run into